src/internet/subsonic/subsonicservice.cpp

```diff
@@ -195,6 +195,10 @@ void SubsonicService::ReloadSettings() {
   username_ = s.value("username").toString();
   password_ = s.value("password").toString();
   usesslv3_ = s.value("usesslv3").toBool();
+  if (s.contains("verifycert"))
```

Comment:
`verifycert_ = s.value("verifycert", true).toBool();` is simpler than this conditional

Reply:
Didn't know it existed, will change ASAP.